src/process/windows/SDL_windowsprocess.c

```diff
@@ -95,6 +95,10 @@ static bool join_arguments(const char * const *args, char **args_out)
             case '"':
                 len += 2;
                 break;
+            case '\\':
+                /* only escape backslashes that precede a double quote */
+                len += (*(a + 1) == '"' || *(a + 1) == '\0') ? 2 : 1;
```

Comment:
I think this reads easier.
```suggestion
                len += (a[1] == '"' || A[1] == '\0') ? 2 : 1;
```